src/sentry/runner/commands/run.py

```diff
@@ -85,7 +86,7 @@ def inner(f):
             "--consumer-group",
             "group_id",
             default=consumer_group,
-            help="Kafka consumer group for the consumer.",
+            help="(physical) Kafka consumer group for the consumer.",
```

Comment:
Does this mean there is now a concept being introduced of a separate logical and physical consumer group name?

Reply:
Yeah that's sort of what i was going for, in the absence of just creating another term. But I'm fine with renaming this concept to something else.